berkeley-mobile/StudyPact/StudyPact.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  StudyPact.swift
+//  berkeley-mobile
+//
+//  Created by Shawn Huang on 2/2/21.
+//  Copyright © 2021 ASUC OCTO. All rights reserved.
+//
+
+import Foundation
+import GoogleSignIn
+
+/// All StudyPact api call functions go in here. Relevant user info also goes here.
+class StudyPact {
+    static let shared = StudyPact()
+    
+    private var cryptoHash: String?
+    private var email: String?
+    var studyGroups: [StudyGroup] = []
```

Comment:
What is this used for?

Reply:
this is for storing the user's info so that it's centralized

Reply:
For `studyGroups` specifically though, it doesn't seem to be used for any purpose currently, and wouldn't we fetch groups from the API instead of storing it here?

Reply:
ah ok yeah I removed studygroups now